src/main/java/org/spongepowered/api/util/Color.java

```diff
@@ -85,33 +107,39 @@
      * @param hex The hexadecimal value of the color
      * @return The color object
      */
-    public static Color ofRgb(int hex) {
-        return ofRgb((hex >> 0x10) & MASK, (hex >> 0x8) & MASK, hex & MASK);
+    public static Color ofHex(int hex) {
+        checkArgument(0 <= hex && hex <= 0xFFFFFF, "hex value [%s] must be in range 0 to 0xFFFFFF (16777215)", hex);
+        return new Color(hex, (hex >> 16) & 0xFF, (hex >> 8) & 0xFF, hex & 0xFF);
     }
 
     /**
      * Gets a new {@link Color} based on the hexadecimal value
      * for a combined {@code red}, {@code green}, and {@code blue}
      * color. Note that colors do not utilize an alpha modifier
      *
-     * @param red The red value
-     * @param green The green value
-     * @param blue The blue value
+     * @param hex The hexadecimal value of the color
      * @return The color object
      */
-    public static Color ofRgb(int red, int green, int blue) {
-        return new Color(red, green, blue);
+    @Deprecated
```

Comment:
If you're deprecating this you need to include a tag in the javadoc stating the alternative.

Reply:
Since you changed a concrete class to an interface, you should simply remove all the deprecated methods scheduled for removal.